docs/dev.md

```diff
@@ -0,0 +1,420 @@
+# Development
+
+You can build, check, and test dcos-net in a development image using
```

Comment:
Could you please move `make docker*` and `make dcos-docker*` to `readme.md` and keep only advanced instruction here? It's pretty easy to start contributing to dcos-net with `make` commands, the steps are really simple and you should know almost nothing about dc/os or erlang infrastructure. So it would be good to advertise them loudly. Advanced instruction is for rare and specific cases.

Reply:
Yes, having the advanced threw me off the tracks a bit, I could have simply used the `dcos-docker*`